### ISSUE TITLE
Freshness scans shows 0 value in the Soda Cloud UI

### ISSUE THREAD

**Issue by sutaraj01** [sodadata/soda-core#1529]
Wanted to notify with v3.0.3 in the cloud I just see a straight line with 0 as the value every time, even though the output of the scans shows the actual time difference.

Scans and screenshots of the result attached. It would be great if it gets resolved quickly, when any failure happens it would be easier to understand from the UI (Even with ms) and not go into the logs to understand how far behind our datasets are. 

```
checks for reservation_browser_debug_v:
  - freshness(res_date, CUST_VAR):
      warn: when > 70m
      fail: when > 150m
```

Edit: Testing on Snowflake datasets

![image](https://user-images.githubusercontent.com/15704934/183396698-a0d88ae8-fd8c-40e3-b11f-e2c9dae50e76.png)
![image](https://user-images.githubusercontent.com/15704934/183396759-95b65a3c-93fe-4df8-af31-ba9ec24f5b8a.png)

**Comment by jmarien** [comment]
SODA-1023

**Comment by vijaykiran** [comment]
Fixed in 3.0.7
